DuckDuckGo/Onboarding/ContextualOnboarding/TrackerMessageProvider.swift

```diff
@@ -165,11 +169,22 @@ struct TrackerMessageProvider: TrackerMessageProviding {
 
         if !currentText.isEmpty {
             let attributes: [NSAttributedString.Key: Any] = isBold ?
-                [.font: NSFont.systemFont(ofSize: fontSize, weight: .bold)] :
-                [:]
+            [.font: boldFont] :
+            [.font: regularFont]
             attributedString.append(NSAttributedString(string: currentText, attributes: attributes))
         }
 
+        let smallString = UserText.ContextualOnboarding.daxDialogTapTheShield
```

Comment:
this snippet below seems similar to the one [OnboardingFireDialogContent](https://github.com/duckduckgo/macos-browser/pull/3391/files#diff-0310cadcb50a995cf59bf3d6d6d5c7c1d93494936c754ed748f073d155b4ff1c) at line 149. Perhaps we can make an helper function to avoid duplicating this code?